vllm/model_executor/models/mixtral.py

```diff
@@ -226,13 +281,6 @@ def __init__(self,
         self.rope_theta = rope_theta
         self.sliding_window = sliding_window
 
-        if isinstance(quant_config, Fp8Config):
```

Comment:
Let's leave this in until we fixed the performance problem with https://github.com/vllm-project/vllm/pull/4118

Reply:
Then we wont be able to load fp8 checkpoints :(

Reply:
Is this still an issue when using static scales? I think it is fairly efficient when reduced to a single static+static gemm enabled by this PR https://github.com/vllm-project/vllm/pull/4332
I could understand if you think the gemms in Mixtral 8x7B are too small for the overhead, but there are larger MoEs

Reply:
Yeah, I believe so -- the problem is with the pytorch scaled_mm function. How about we leave the if in and change it to 
```python
if isinstance(quant_config, Fp8Config) and not quant_config.is_checkpoint_fp8_serialized:
```
That way we have a high-performance codepath for people that care about it like us (until the performance is fixed, then we can remove it).

Reply:
I'm good with disabling it for the dynamic quant case, thanks